hw/ip/rv_core_ibex/data/rv_core_ibex.hjson

```diff
@@ -806,6 +813,25 @@
           },
         ]
       },
+
+      { name: "FPGA_INFO",
+        desc: '''
+          FPGA build timestamp info.
+          This register only contains valid data for fpga, for all other variants it is simply 0.
+        ''',
+        swaccess: "ro",
```

Comment:
Could choose to make this a rw scratch register for the ASIC, but perhaps it's best to leave it as also encoding whether we have an FPGA or not. (unless there is some other ID that already gives that indication)

Reply:
yeah i thought aobut this a bit. I think i'll leave this for now. Since i encoded this as hwext, it's actually "empty" space.